conf.py

```diff
@@ -59,7 +59,7 @@
 # The short X.Y version.
 version = '7.0'
 # The full version, including alpha/beta/rc tags.
-release = '7.0.2'
+release = '7.0.3'
```

Comment:
I don't really know why we track the patch version in docs like this, but we've been doing it that way so I guess we can continue doing so. Maybe at the next major version we can make `version` and `release` always match.